flow-defs/src/utils.js.flow

```diff
@@ -32,6 +33,7 @@ declare export var updateNavigationBar: (x: {
 declare export var setWebViewTitle: (title: string) => Promise<void>;
 declare export var notifyPageLoaded: () => Promise<void>;
 declare export var notifyBridgeReady: () => Promise<void>;
+declare export var getRemoteConfig: () => Promise<RemoteConfig>;
```

Comment:
we should remove flow types generation, I've created a ticket https://jira.tid.es/browse/WEB-774